src/components/shared/Page/Page.stories.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import {storiesOf} from '@storybook/react';
 import {withKnobs, text} from '@storybook/addon-knobs';
 import Page from './Page';
-import LastLessonProvider from '../../../providers/LastLessonProvider';
```

Comment:
More-so for my own knowledge, why are we removing LastLessonProvider?

Reply:
It was never working to begin with, and was causing issues with some of my tests. I can't remember when it ever actually worked, so I'm removing it. Also, it's a GDPR issue if we are setting local storage and not asking users for permission to do so, since it's not essential/core functionality to the site.